build.sh

```diff
@@ -49,3 +48,13 @@ case "$os" in
     mkdir -p ${OUTDIR}
     cp -f build/bin/Datadog.Trace.ClrProfiler.Native.${SUFIX} ${OUTDIR}/OpenTelemetry.AutoInstrumentation.ClrProfiler.Native.${SUFIX}
 esac
+
+# build Managed
+cd $DIR
+OUTDIR="$( pwd )/src/Datadog.Trace.ClrProfiler.Native/bin/${BUILD_TYPE}/x64"
```

Comment:
maybe move the 
```
OUTDIR="$( pwd )/src/Datadog.Trace.ClrProfiler.Native/bin/${BUILD_TYPE}/x64"
```
declaration before building native dll's

and remove `OUTDIR="bin/${BUILD_TYPE}/x64"` as it may lead to tedious bugs if someone changes only one delcation by mistake